### PR TITLE
6 get comment by id

### PR DESCRIPTION
**Pull request by ElenaCKay**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -113,3 +113,50 @@ describe("GET /api/reviews", () => {
             });
     });
 });
+
+describe.only("GET /api/reviews/:review_id/comments", () => {
```
Comment (HeatherEver) [inline __tests__/app.test.js]: remove .only before submitting prs in future

File: __tests__/app.test.js [hunk 2]
```diff
@@ -113,3 +113,50 @@ describe("GET /api/reviews", () => {
...
+                expect(body.msg).toBe("Invalid ID");
+            });
+    });
+    test("404: GET response for a valid but non existant review ID", () => {
+        return request(app)
+            .get("/api/reviews/1000/comments")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Not found");
+            });
```
Comment (HeatherEver) [inline __tests__/app.test.js:169]: could you please add a 200 test for an article that has no comments

File: controllers/reviews.controller.js [hunk 3]
```diff
@@ -13,10 +13,23 @@ exports.getReviewById = (req, res, next) => {
...
+            res.status(200).send({ reviews });
+        })
+        .catch((err) => {
+            next(err);
+        });
+};
+
+exports.getCommentsById = (req, res, next) => {
+    const { review_id } = req.params;
+    console.log(review_id)
```
Comment (HeatherEver) [inline controllers/reviews.controller.js]: remove console logs

**Review by HeatherEver — APPROVED** [review]
Nice work here, just one more test to add which might mean a little refactor elsewhere too :)

File: __tests__/app.test.js [hunk 4]
```diff
@@ -113,3 +113,59 @@ describe("GET /api/reviews", () => {
...
+                const { comments } = body;
+                expect(comments).toHaveLength(3);
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        review_id: expect.any(Number),
```
Comment (Poonam-raj) [inline __tests__/app.test.js:132]: You should definitely assert a specific review_id here as you know all comments should belong to one article.

**Review by Poonam-raj — APPROVED** [review]
Very very nice work thanks for amending 🎉 
Only one change to make with a assertion in your testing but once corrected pls feel free to merge 🚀

**Pull request merged**